Course_Part_10/src/TicTacToeBasic.java

```diff
@@ -0,0 +1,247 @@
+import java.sql.Array;
+import java.util.*;
+
+
+public class TicTacToeBasic {
+
+    static TreeMap<String, String> playOptions;
+    static Scanner input = new Scanner(System.in);
+    static String playMode;
+
+    public static void main(String[] args) {
+
+        playOptions = new TreeMap<>() {{
```

Comment:
regel 13 t/m 28 kun je in een methode zetten genaamd initGame() of iets dergelijks

Reply:
Goed idee.